hub/components/NewWallet/useNewWalletTransaction.ts

```diff
@@ -79,10 +84,25 @@ const useNewWalletCallback = (
       wallet.publicKey,
     );
 
+    // createTicketIxs is a list of instructions that create nftActionTicket only for nft-voter-v2 plugin
+    // so it will be empty for other plugins
+    const createTicketIxs = instructions.slice(0, -3);
```

Comment:
this kind of thing is a big no-no. makes the code really fragile, since adding instructions will break this. i understand why you did it but the old way was better than this. my comments before about changing the API were before I understood the parallel vs sequential thing. just do it how you were doing before. it's not your fault everything sucks. but yeah this is way too fragile.